readme updates (#245)
leolost2605/SBAid: README.md

# SBAid

> SBAid is a tool allowing for easy testing of dynamic freeway control systems (**S**trecken**b**eeinflussungs**a**nlagen in german) in a simulation.

Reviewer: ```suggestion
SBAid is a tool allowing for easy testing of dynamic freeway control systems (german: **S**trecken**b**eeinflussungs**a**nlagen) in a simulation.
```

SBAid ist written in python using GTK with libadwaita as its UI toolkit. We highly recommend
taking a look at the [PyGObject](https://pygobject.gnome.org/) documentation before getting started.

## File input

> SBAid needs file inputs for many of the app's main functionalities. Follow the given guidelines for a problem-free interaction with file inputting in SBAid:

Reviewer: ```suggestion
SBAid needs file inputs for many of the app's main functionalities. Follow the given guidelines for problem-free file input in SBAid:
```
